src/components/organize/tasks/forms/TaskDetailsForm.tsx

```diff
@@ -96,6 +96,7 @@ const TaskDetailsForm = ({ onSubmit, onCancel, task }: TaskDetailsFormProps): JS
                 title: task?.title,
                 type: task?.type,
             }}
+            keepDirtyOnReinitialize
```

Comment:
Interesting! Good find. 👍

Reply:
yea the docs are pretty obscure too, it was a bit of a hunt